changelog/288.txt

```diff
@@ -1,3 +1,3 @@
 ```release-note:improvement
-runners: The host IPTables runner now takes an optional Timeout value, so that long-running executions can be gracefully stopped.
+documentation: add note about ubuntu installation requiring root privileges, and demonstrate switching user in the example commands
```

Comment:
I think maybe there was a file copied but not renamed. 😄

Reply:
Sorry, I *severely* mis-read a directory listing. Twice.